### PR TITLE
Issue1649 delay 200 ok to invite

### PR DESCRIPTION
**Pull request by kacmos**
George,
please let me know your opinion about modification.

BR
Kacper

### CODE REVIEW

**Comment by gvagenas** [comment]
Hi @kacmos , thanks for the PR , will review and update you asap

**Comment by gvagenas** [comment]
Hi @kacmos , LiveCallModification tests are failing

**Comment by kacmos** [comment]
Hi,
I will take a look on LCM tests

**Comment by gvagenas** [comment]
@kacmos check org.restcomm.connect.testsuite.http.LiveCallModificationTest.
Let me know if you want to have a conf call to discuss this

File: restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/VoiceInterpreter.java [hunk 1]
```diff
@@ -2087,6 +2110,9 @@ public void execute(final Object message) throws Exception {
                 }
                 record(bridge);
             }
+            if(enable200OkDelay){
```
Comment (gvagenas) [inline restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/VoiceInterpreter.java]: @kacmos even thought with this LCM is working when enable200OKDelay == false, did you checked if LCM will work when the feature will be true?
I think LCM wont work when enable200OKDelay == true.
We need to find a better way to protect LCM

**Comment by gvagenas** [comment]
Thanks for the update @kacmos , will check now

**Comment by kacmos** [comment]
@gvagenas please take a look on my correction for LCM

**Comment by gvagenas** [comment]
@kacmos I checked the new patch but I still can see that the LCM tests for answer delay org.restcomm.connect.testsuite.http.LiveCallModificationAnswerDelayTest are failing.

Still checking that to find out the cause

**Comment by gvagenas** [comment]
@kacmos after your last commit, all LCM tests are passing, I continue review and test the rest. I will update here

**Comment by gvagenas** [comment]
@kacmos PR reviewed and merged, thanks

**Comment by peteroyle** [comment]
Hi @gvagenas and @kacmos, we have been very much looking forward to this! Thank you for your efforts :) 👍

**Pull request merged**